docs/build-and-publish-pipeline.md

```diff
@@ -28,6 +28,10 @@ The key points:
         * Publish it to the [maven.mozilla.org](https://maven.mozilla.org).
 * Notifications about build failures are sent to a mailing list at
   [a-s-ci-failures@mozilla.com](https://groups.google.com/a/mozilla.com/forum/#!forum/a-s-ci-failures)
+* Taskcluster scopes are managed by the Release Engineering team using `ciadmin`. The proper way to
```

Comment:
Nit, not blocking: maybe it's worth making the `ciadmin` word a href to point to https://hg.mozilla.org/ci/ci-admin/ ?